packages/mwp-app-render/src/components/PageWrap/index.jsx

```diff
@@ -40,7 +40,7 @@ class PageWrap extends React.Component {
 			});
 
 			// Add a custom attribute to the PageView & BrowserInteraction events in Insights
-				window.newrelic.setCustomAttribute('timeToAppInteractive', now);
+span class="pl-c1"> 			window.newrelic.setCustomAttribute('timeToAppInteractive', now);
```

Comment:
Arrgh, looks like they have a bug there and this code leaks into the diff

Reply:
yeah, it's still in there. i'll do the commit manually through my editor